src/sentry/models/organizationoption.py

```diff
@@ -0,0 +1,126 @@
+"""
+sentry.models.organizationoption
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import, print_function
+
+from celery.signals import task_postrun
+from django.core.signals import request_finished
+from django.db import models
+
+from sentry.db.models import Model, FlexibleForeignKey, sane_repr
+from sentry.db.models.fields import UnicodePickledObjectField
+from sentry.db.models.manager import BaseManager
+from sentry.utils.cache import cache
+
+
+class OrganizationOptionManager(BaseManager):
+    def __init__(self, *args, **kwargs):
+        super(OrganizationOptionManager, self).__init__(*args, **kwargs)
+        self.__cache = {}
+
+    def __getstate__(self):
+        d = self.__dict__.copy()
+        # we cant serialize weakrefs
+        d.pop('_OrganizationOptionManager__cache', None)
+        return d
+
+    def __setstate__(self, state):
+        self.__dict__.update(state)
+        self.__cache = {}
+
+    def _make_key(self, instance_id):
+        assert instance_id
+        return '%s:%s' % (self.model._meta.db_table, instance_id)
+
+    def get_value_bulk(self, instances, key):
+        instance_map = dict((i.id, i) for i in instances)
+        queryset = self.filter(
+            organization__in=instances,
+            key=key,
+        )
+        result = dict((i, None) for i in instances)
+        for obj in queryset:
+            result[instance_map[obj.organization_id]] = obj.value
```

Comment:
This is a copy/paste/replace from `sentry/models/projectoption.py`, but this `obj.organization_id` attribute seems wrong?